celerymon/cli/__init__.py

```diff
@@ -60,6 +65,17 @@ def healthz_wrapper(
         httpd.serve_forever()
 
 
+def parse_histogram_buckets(arg: T.Optional[str]) -> T.Sequence[T.Union[float, str]]:
```

Comment:
Is this type correct? Seems(?) like it should always bee `Sequence[float]`

Reply:
This returns `Sequence[float]` but the buckets are defined as `Sequence[Union[float, str]]`, so taking that as the retval.

https://github.com/prometheus/client_python/blob/master/prometheus_client/metrics.py#L555

Reply:
Maybe I can just simplify this considering that prom client does most of the job.